src/components/mdx/Code.tsx

```diff
@@ -17,25 +17,32 @@ require('prismjs/components/prism-clojure')
 
 // Used to determine if we have to highlight the given index
 // Borrowed from https://prince.dev/blog/highlight-with-react
-const calculateLinesToHighlight = meta => {
+const calculateLinesToHighlight = (meta: string) => {
   const RE = /{([\d,-]+)}/
   if (RE.test(meta)) {
-    const strlineNumbers = RE.exec(meta)[1]
+    const strlineNumbers = RE.exec(meta)?.[1] ?? ''
```

Comment:
`exec` might return `null` so I added:

1. `?.[1]` to prevent TypeError
2. `?? ''` to always have a string

WDYT?

Reply:
That sounds good to me! I think I copied this from my friend a while back lol